packages/blast-utilities/contract-utils.js

```diff
@@ -9,29 +9,25 @@ const BlastError = require('./blast-error')
 const { getProjectRootPath } = require('./package-info')
 
 module.exports.CudosContract = class CudosContract {
-  #contractName
+  #contractLabel
```

Comment:
why is ti changed from name to label?

Reply:
I changed it for consistency with the "label" we are getting from CosmWasmClient in globals.js in the "getContractFromAddress" function where we get a contract object with a label from an already deployed contract